site/index.html

```diff
@@ -112,28 +112,29 @@
         <header>
           <h1 class='text-red'>Typography</h1>
         </header>
-        <h1>H1 ITC Avant Garde Gothic - Bold 39px</h1>
-        <h2>H2 ITC Avant Garde Gothic - Bold 31px</h2>
-        <h3>H3 ITC Avant Garde Gothic - Bold 25px</h3>
-        <h4>H4 ITC Avant Garde Gothic - Bold 20px </h4>
-        <p>Noto Sand Regular 16px</p>
-        <p class='font-bold'>Noto Sand Regular 16px</p>
-        <p class='text-gray-lighter'>Noto Sand Regular 16px</p>
-        <p class='text-sm'>notes 12.8px</p>
-        <small>more notes in small</small>
+        <h1>H1 Raleway - Black 40px</h1>
+        <h2>H2 Raleway - Black 32px</h2>
+        <h3>H3 Raleway - Black 24px</h3>
+        <h4>H4 Raleway - Black 20px </h4>
+        <p>p Poppins Regular 16px</p>
+        <p class='font-bold'>p_bold Poppins Bold 16px</p>
```

Comment:
Is that text correct `p_bold`?  What does it mean, or why is it there?

Reply:
p_bold is a single text to reference the style of `font-bold` class in `p` tags, we are using the new design-system as reference to make this changes.

[new design system](https://www.figma.com/file/9jEtKaJ8JhC5QcB0bqWBAo/NordenDS?node-id=0%3A1)